includes/data.php

```diff
@@ -1767,7 +1769,7 @@ function pods_serial_comma( $value, $field = null, $fields = null, $and = null,
 				if ( null !== $params->field_index && isset( $v[ $params->field_index ] ) ) {
 					$v = $v[ $params->field_index ];
 				} elseif ( $simple ) {
-					$v = trim( implode( $params->separator . ' ', $v ), $params->separator . ' ' );
+					$v = trim( implode( $params->separator, $v ), $params->separator );
```

Comment:
@sc0ttkclark Any idea why it's wrapped in a trim() function? I don't see why it's needed but wanted to verify before removing it. Same thing on line 1783.

Reply:
It trims() any extra spaces/separators already in the value text

Reply:
```suggestion
					$v = trim( implode( $params->separator . ' ', $v ), $params->separator . ' ' );
```